packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -157,44 +146,18 @@ export const FileSnapshotWriterClassFactory = <TBase extends ReaderConstructor>(
                 this.docId = versionId;
             }
 
-            if (versionId !== null && this.commitsWriter[versionId] !== undefined) {
```

Comment:
This is great. All this code you are removing is unnecessary and I am happy to see it gone :-)

Reply:
@vladsud as FYI. Maybe these were added because some option in replay tool needs it?

Reply:
The `storageService` writes I have seen were around: `this.storageService.write(root, parents, message, "");`. `ref`-s were empty strings which would imply `commitsWriter` did not play any further role. But yes, please share anything I may have missed.

Reply:
@vladsud if you have no further comments, I'll resolve this convo.